build.sbt

```diff
@@ -23,6 +23,8 @@ val slickV = "3.1.1"
 
 val googleClientApiV = "1.20.0"
 
+val googleGenomicsVersion = "v1alpha2-rev7-1.19.1"
```

Comment:
@tovanadler Are you sure?  I see `v1alpha2-rev8-1.21.0` at the following location:

https://oss.sonatype.org/#nexus-search;quick~google-api-services-genomics

The direct download link is the following:

https://oss.sonatype.org/service/local/repositories/releases/content/com/google/apis/google-api-services-genomics/v1alpha2-rev8-1.21.0/google-api-services-genomics-v1alpha2-rev8-1.21.0.jar

~p